decidim-budgets/app/helpers/decidim/budgets/projects_helper.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+module Decidim
+  module Budgets
+    # A helper to render order and budgets actions
+    module ProjectsHelper
+      # Render a budget as a currency
+      #
+      # budget - A integer to represent a budget
+      def budget_to_currency(budget)
+        number_to_currency budget, unit: "€", delimiter: ".", precision: 0, format: "%n %u"
```

Comment:
This should come from the locales. See [the English locales file](https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/en.yml#L155-L163) for an example.